package.json

```diff
@@ -29,7 +29,7 @@
     "glob": "7.1.2",
     "gulp-filter": "5.1.0",
     "insight": "0.10.1",
-    "jhipster-core": "3.0.2",
+    "jhipster-core": "3.6.8",
```

Comment:
I'm doing a PR to update the generator, and bump JCore's version to v3.6.11

Reply:
:+1: Thanks a lot @MathieuAA!!!